packages/extension-base/src/background/handlers/Extension.ts

```diff
@@ -177,6 +177,51 @@ export default class Extension {
     return true;
   }
 
+  private jsonRestore ({ json, password }: RequestJsonRestore): ResponseJsonRestore {
+    try {
+      const pair = keyring.restoreAccount(json, password);
+
+      if (pair) {
+        const { address } = pair;
+
+        return { message: `Successfully added ${address}` };
+      }
+    } catch (error) {
+      return { message: `${error}` };
```

Comment:
```suggestion
      return { message: (error as Error).message };
```

(lint fix. the issue is that in JS a `throw` can be anything, even a string, but since it is only our stuff, we know these are always an `Error`)